aicsimageio/aics_image.py

```diff
@@ -305,6 +305,16 @@ def metadata(self) -> Any:
         # The reader can implement read optimization or not.
         return self.reader.metadata
 
+    @property
```

Comment:
I think you don't want to name a property as "get_something".   A property would generally have a noun name and the get is implied when you access it.  I think this is maybe more like a function, to indicate that it has to do some conversion each time.

Reply:
Also, shouldn't all these properties be marked as read only somehow?  I am afraid we are abusing `@property` in this lib

Reply:
Strong agree. This shouldn't be a property. I think the reason we overload properties on the repo a lot is because we do the lazy loading behavior. "Once it's loaded, hold onto it"

Reply:
Ok, thanks for making this point. I was thinking of properties as functions that didn't take arguments. I'll change it back to a function. In that case are you guys good with the naming or would you like to see it changed?

Reply:
I personally prefer `get_ome_metadata` but the current name also works.